modules/oe_theme_helper/tests/src/Unit/TwigExtensionTest.php

```diff
@@ -300,4 +303,110 @@ public function toEclIconProvider(): array {
     ];
   }
 
+  /**
+   * Tests trimming of textfield.
+   *
+   * @param mixed $textfield
+   *   The string, object or render array.
+   * @param array $context
+   *   The context array.
+   * @param mixed $expected_result
+   *   The expected result.
+   *
+   * @covers ::trimTextfield
+   * @dataProvider trimTextfieldProvider
+   */
+  public function testTrimTextfield($textfield, array $context, $expected_result): void {
+    $context['input_data'] = $textfield;
+    $result = $this->twig->render("{{ trim_textfield(input_data)|json_encode() }}", $context);
```

Comment:
Call the function using the $textfield from the data provider (as the function uses the parameter) instead of setting it in the $context.

Reply:
Talked to Sergii and the result would not be the same.